cmd/manager/main.go

```diff
@@ -20,6 +20,9 @@ import (
 	"flag"
 	"os"
 
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/apis"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/aws/actuators/cluster"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/aws/actuators/machine"
```

Comment:
Not a blocker by any means, but I think there is value in keeping the local imports in a separate group.

Reply:
Looking at the ordering applied there, that might be `gofmt`, which means that local imports should always be grouped together at least, even if not separate.